package.json

```diff
@@ -143,19 +144,19 @@
     "webdriver-manager": "^12.1.8",
     "webpack": "^3.5.4",
     "webpack-bundle-analyzer": "^3.0.2",
-    "webpack-dev-server": "2.11.3",
-    "@okta/eslint-plugin-okta": "0.4.0"
+    "webpack-dev-server": "2.11.3"
   },
   "dependencies": {
     "@babel/polyfill": "^7.10.1",
     "@babel/runtime": "^7.10.3",
     "cross-fetch": "^3.0.4",
     "handlebars": "^4.5.3",
+    "jquery": "^3.6.0",
```

Comment:
seems not necessary to be public dependence given we bundle everything ?

Reply:
Adding onto the comment above, does this change our [multiple output configurations](https://github.com/okta/okta-signin-widget/#using-the-npm-module)?

Reply:
The convention that I follow is to include all code which is part of the bundle in dependencies. When we publish as an ESM package this will be important. I also think it helps for security audits to know whether something is a "dev" dependency (tooling, tests) vs. a production dependency (code exists in the bundle and is shipped to end-users). @jmelberg-okta this has no effect on the output, but dependencies are installed along with the module so it would increase install time slightly.